File: output/qwen_2.5_72b_instruct/avatar/algo_based_translation/python/java/successful_translation/codeforces_546_A.java

import java.util.Scanner;

public class codeforces_546_A {
    public static int calculateShortage(int k, int n, int w) {
        int total = 0;
        for (int i = 1; i <= w; i++) {
            total += i * k;
        }
        if (total > n) {
            return total - n;
        } else {
            return 0;
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] values = input.split(" ");
        int k = Integer.parseInt(values[0]);
        int n = Integer.parseInt(values[1]);
        int w = Integer.parseInt(values[2]);
        
        if (1 <= k && w <= 1000) {
            System.out.println(calculateShortage(k, n, w));
        }
        scanner.close();
    }
}

File: dataset/avatar/Python/Code/codeforces_546_A.py
k , n , w = input ( ).split ( )
if 1 <= int ( k ) and int ( w ) <= 1000 :
    total = 0
    for i in range ( 1 , int ( w ) + 1 ) :
        total += i * int ( k )
    if total > int ( n ) :
        print ( total - int ( n ) )
    else :
        print ( 0 )

